src/main/java/mate/academy/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.dao.impl;
+
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+import mate.academy.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order save(Order order) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(order);
+            transaction.commit();
+            return order;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't insert an order : " + order, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+    }
+
+    @Override
+    public List<Order> getByUser(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Order> query = session.createQuery("from Order o "
```

Comment:
It is better to write all SQL commands in upper case
```suggestion
            Query<Order> query = session.createQuery("FROM Order o "
```